internal/pkg/term/progress/ecs.go

```diff
@@ -180,6 +200,98 @@ func (c *rollingUpdateComponent) renderAlarms(out io.Writer) (numLines int, err
 	return renderComponents(out, components)
 }
 
+type stoppedTasksInfo struct {
+	ids              []string
+	latestStoppingAt time.Time
+}
+
+func (c *rollingUpdateComponent) renderStoppedTasks(out io.Writer) (numLines int, err error) {
+	if len(c.stoppedTasks) == 0 {
+		return 0, nil
+	}
+	header := []string{"TaskId", "CurrentStatus", "DesiredStatus"}
+	var rows [][]string
+	title := fmt.Sprintf("Latest %d %s stopped reason", len(c.stoppedTasks), english.PluralWord(len(c.stoppedTasks), "task", "tasks"))
+	title = fmt.Sprintf("%s%s", color.DullRed.Sprintf("✘ "), color.Faint.Sprintf(title))
+	childComponents := []Renderer{
+		&singleLineComponent{}, // Add an empty line before rendering task stopped events.
+		&singleLineComponent{
+			Text:    title,
+			Padding: c.padding,
+		},
+	}
+
+	taskInfoMap := make(map[string]stoppedTasksInfo, len(c.stoppedTasks))
+	for _, st := range c.stoppedTasks {
+		id, err := ecs.TaskID(aws.StringValue(st.TaskArn))
+		if err != nil {
+			return 0, err
+		}
+		existingTaskInfo, ok := taskInfoMap[aws.StringValue(st.StoppedReason)]
+		if ok {
+			existingTaskInfo.ids = append(existingTaskInfo.ids, ecs.ShortTaskID(id))
+			existingTaskInfo.latestStoppingAt = aws.TimeValue(st.StoppingAt)
+			taskInfoMap[aws.StringValue(st.StoppedReason)] = existingTaskInfo
+		} else {
+			taskInfoMap[aws.StringValue(st.StoppedReason)] = stoppedTasksInfo{
+				ids:              []string{ecs.ShortTaskID(id)},
+				latestStoppingAt: aws.TimeValue(st.StoppingAt),
+			}
+		}
+		rows = append(rows, []string{
+			ecs.ShortTaskID(id),
+			aws.StringValue(st.LastStatus),
+			aws.StringValue(st.DesiredStatus),
+		})
+	}
+	var sortReasons []string
+	for reason := range taskInfoMap {
+		sortReasons = append(sortReasons, reason)
+	}
+	sort.SliceStable(sortReasons, func(i, j int) bool {
+		return taskInfoMap[sortReasons[i]].latestStoppingAt.After(taskInfoMap[sortReasons[j]].latestStoppingAt)
+	})
+	for _, reason := range sortReasons {
+		for i, truncatedReason := range splitByLength(fmt.Sprintf("[%s]: %s", strings.Join(taskInfoMap[reason].ids, ","), reason), maxCellLength) {
+			pretty := fmt.Sprintf("  %s", truncatedReason)
+			if i == 0 {
+				pretty = fmt.Sprintf("- %s", truncatedReason)
+			}
+			childComponents = append(childComponents, &singleLineComponent{
+				Text:    pretty,
+				Padding: c.padding + nestedComponentPadding,
+			})
+		}
+	}
+	table := newTableComponent(color.Faint.Sprintf("Latest %d stopped %s", len(c.stoppedTasks), english.PluralWord(len(c.stoppedTasks), "task", "tasks")), header, rows)
+	table.Padding = c.padding
+	childComponents = append(childComponents,
+		&singleLineComponent{},
+		&singleLineComponent{
+			Text:    color.Faint.Sprintf("Troubleshoot task stopped reason"),
+			Padding: c.padding,
+		},
+		&singleLineComponent{
+			Text: fmt.Sprintf("1. You can run %s to see the logs of the last stopped task.",
```

Comment:
quick question - should we _always_ suggest this, or should we suggest this only if a stopped reason is "essential container exit" 🤔 what do you think?

Reply:
According to this [doc ](https://repost.aws/knowledge-center/ecs-task-stopped), incase of `ELB health check failure` there are suggesting to check for the application logs.

Also there might be  a scenario where they can have print/log statements in the application code, in this case `copilot svc logs --previous` might be helpful. So I am thinking to include this recommendation for all the cases.